src/pages-and-resources/live/data/api.js

```diff
@@ -0,0 +1,165 @@
+import { ensureConfig } from '@edx/frontend-platform';
+import { getAuthenticatedHttpClient } from '@edx/frontend-platform/auth';
+
+ensureConfig([
+  'STUDIO_BASE_URL',
+], 'Course Apps API service');
+
+function normalizeLtiConfig(data) {
+  if (!data || Object.keys(data).length < 1) {
+    return {};
+  }
+
+  return {
+    consumerKey: data.lti_1p1_client_key,
+    consumerSecret: data.lti_1p1_client_secret,
+    launchUrl: data.lti_1p1_launch_url,
+  };
+}
+
+function normalizePiiSharing(data) {
+  return {
+    piiSharing: data.pii_sharing_allowed,
+    piiShareUsername: data.pii_sharing_allowed,
+    piiShareEmail: data.pii_sharing_allowed,
```

Comment:
if all these value are now extracted from a single source of truth from backend why we still need three different identifiers on frontend?

Reply:
I'm not sure the exact structure of API response.  This code will be more clean on integration time.